src/lib/utils.ts

```
@@ -1,6 +1,5 @@
-import type { ClassValue } from 'clsx'
-import { clsx } from 'clsx'
-import { twMerge } from 'tailwind-merge'
+import { clsx, type ClassValue } from "clsx"
```

ESLint

[error] Prefer using a top-level type-only import instead of inline type specifiers.

(import/consistent-type-specifier-style)